src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -4167,6 +4167,94 @@ instruct scatterL(indirect mem, vReg src, vReg idx) %{
   ins_pipe(pipe_slow);
 %}
 
+// ------------------------------ Vector Load Gather Partial-------------------------------
+instruct gatherI_partial(vReg dst, indirect mem, vReg idx, pRegGov pTmp, rFlagsReg cr) %{
+  predicate(UseSVE > 0 &&
+            n->as_LoadVectorGather()->memory_size() >= 16 &&
```

Comment:
Since there is the min vector size (size in byte >= 16) limitation in `op_sve_supported`, is this needed here?

Reply:
Sure. Thank you for your review. I will change that.